Raids/Prophecy of Ahnket/Normal/Tartaric_Depths/TarJulia.lua

```diff
@@ -301,12 +306,15 @@ function TAR:Start()
 
     KBM.Defaults.TimerObj.Assign(self.TarJulia)
 
-    self.TarJulia.MechRef.SpiderWeave = KBM.MechSpy:Add(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, "playerDebuff", self.TarJulia)
-    KBM.Defaults.MechObj.Assign(self.TarJulia)
-
     -- Create Alerts
     self.TarJulia.AlertsRef.MoltenBlast = KBM.Alert:Create(self.Lang.Verbose.MoltenBlast[KBM.Lang], 3, true, true, "red")
+    self.TarJulia.AlertsRef.CallForHelp = KBM.Alert:Create(self.Lang.Ability.CallForHelp[KBM.Lang], nil, true, true, "blue")
     self.TarJulia.AlertsRef.Fury = KBM.Alert:Create(self.Lang.Buff.Fury[KBM.Lang], 1, true, true, "orange")
+    self.TarJulia.AlertsRef.SpiderWeave = KBM.Alert:Create(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, true, true, "purple")
+    self.TarJulia.MechRef.SpiderWeave = KBM.MechSpy:Add(self.Lang.Debuff.SpiderWeave[KBM.Lang], nil, "playerDebuff", self.TarJulia)
```

Comment:
I prefer to have thing in that order: 
  - timers
  - mechspy
  - alerts
  - triggers